### ISSUE TITLE
ABAP CDS outdated

### ISSUE THREAD

**Issue by gabrielbaca** [FreHu/abap-cds-grammar#1]
Hello!

It seems that ABAP CDS has continued to evolve, but the grammar in here has not been updated. Keywords such as **root**, **composition** and **association to parent** are not highlighted (which are all associated to RAP). 

I can create a pull request and add the ones I've identified, but I've noticed there is another repository of yours @FreHu that contains a more recent version at [vscode-abap-cds](https://github.com/SAP-samples/vscode-abap-cds/blob/master/syntaxes/cds.tmLanguage.json). Having a quick look the JSON seems very similar, is that something that can be replicated here?

**Comment by FreHu** [comment]
Hi! 

This is indeed supposed to be a copy of the grammar in vscode-abap-cds, I've updated them to match.

why there are two repositories:

I have this repo separately because of licensing issues with my linguist submission which were not related to the grammar, but the snippets contained in the vscode extension.

I made the linguist submission a long time ago and never actually expected it to be approved, but recently, the `asddls` extension was approved due to rising popularity. Now the github syntax highlighting is derived from this grammar.

Feel free to submit any missing keywords you find, I'll sync the repositories again and update the extension. Note that changes to github highlighting will only become visible once a new version of linguist is released. If you want to test the github colors, it can be done here https://github-lightshow.herokuapp.com/.
